sdk/eventhub/event-hubs/README.md

```diff
@@ -138,142 +138,108 @@ For example: `body: { "message": "Hello World" }`
 
 ### Consume events from an Event Hub partition
 
-To consume events from a single Event Hub partition in a consumer group, create an `EventHubConsumer` for that partition and consumer group combination. You will need to provide a position in the event stream from where to begin receiving events; in our example, we will read new events as they are published.
+To consume events from a single Event Hub partition in a consumer group, create an `EventHubConsumerClient` for that partition and consumer group combination. You will need to provide a position in the event stream from where to begin receiving events; in our example, we will read new events as they are published.
 
 ```javascript
-const client = new EventHubClient("connectionString", "eventHubName");
-const consumer = client.createConsumer(
+const client = new EventHubConsumerClient("connectionString", "eventHubName");
+const subscription = client.subscribe(
   EventHubClient.defaultConsumerGroupName,
-  partitionIds[0],
-  EventPosition.latest()
+  (receivedEvents, context) => { },
+  [ partitionIds[0] ], {
```

Comment:
Can you update the sample to also show how a user could get the partitionIds?

Reply:
Done